### ISSUE TITLE
Update to current materialdesignicons version please

### ISSUE THREAD

**Issue by shynline** [code-mc/material-icon-lib#55]

**Comment by T-bond** [comment]
Yeah, the library is a bit outdated now. It would be cool if it could get an update, as I am missing a lot of the icons from my application.

**Comment by saeed74** [comment]
end of 2020 and still waiting for update

**Comment by saeed74** [comment]
2022 and I'm still waiting!

**Comment by shynline** [comment]
No activity from the maintainer in their Github.
Probably a dead project.
Maybe It's better for someone to take responsibility and fork this project and become a new maintainer. 
Compose support would be nice as well :).
